README.md

```diff
@@ -114,3 +114,26 @@ This specification aims to be:
 ### Spelling
 
 Attempt to use descriptive English words (or _very common_ abbreviations) in documentation and identifiers. If necessary, there is a custom dictionary `whitelist.txt`.
+
+
+## CLI Utilities `ethereum_spec_tools`
+
+### New Fork Tool
+This tool can be used to creates the base code for a new fork by using the existing code from a given fork.
+
+The command takes 4 arguments, 2 of which are optional
+ * from_fork: The fork name from which the code is to be duplicated. Eg. - "Tangerine Whistle"
+ * to_fork: The fork name of the new fork Eg - "Spurious Dragon"
+ * from_test (Optional): Name of the from fork within the test fixtures in case it is different from fork name. Eg. - "EIP150"
+ * to_test (Optional): Name of the to fork within the test fixtures in case it is different from fork name Eg - "EIP158"
+
+As an example, if one wants to create baseline code for the `Spurious Dragon` fork from the `Tangerine Whistle` one
+
+```bash
+python src/ethereum_spec_tools/new_fork.py --from_fork="Tangerine Whistle" --to_fork="Spurious Dragon" --from_test=EIP150 --to_test=EIP158
+```
```

Comment:
Oh! This reminds me, the tool should also be added [in `setup.cfg`](https://github.com/ethereum/execution-specs/blob/7149615e00d8206a531ab601bca4e14c57b0535b/setup.cfg#L80).

Reply:
Added